draft-ietf-quic-multipath.md

```diff
@@ -1234,6 +1207,9 @@ Reason Phrase:
 PATH_ABANDON frames are ack-eliciting. If a packet containing
 a PATH_ABANDON frame is considered lost, the peer SHOULD repeat it.
 
+Once a Path ID is retired by PATH_ABANDON,
+it MUST NOT be reused on any other path.
```

Comment:
The current formulation is a bit implicit about what is meant by "reused". What about this? In my opinion, we could also drop the "even on other network paths" part.
```suggestion
endpoints MUST NOT send packets associated with that Path ID anymore,
even on other network paths.
```